EIPS/eip-6969.md

```diff
@@ -0,0 +1,169 @@
+---
+eip: 6969
```

Comment:
This PR was opened to reserve the meme number. The first commit (3a485ae6300ab8f224ebb0e6179269f43ad06c7e) is just a preamble and the EIP template. The EIP remained mostly empty until May 6th (6c7f200a8f91451fe8eedb730221a1eb57b939df).

We do our best to discourage number sniping, and @abcoathup has volunteered to reassign such numbers as they arise.

To be perfectly clear, if this proposal had content at the time it was pull requested, we wouldn't be reassigning this number.

```suggestion
eip: 6968
```

Reply:
Change has been submitted. Can we get this merged now? Thank you!

Reply:
The automatic validator is still flagging some issues that need to be fixed.
https://github.com/ethereum/EIPs/pull/6969/files

For example:
* discussions-to should point to the Ethereum magicians thread
* add your GitHub handle as an author
* There are also a number of "TODO Zak"